compiler/rustc_hir_analysis/src/astconv/bounds.rs

```diff
@@ -565,7 +565,7 @@ fn check_assoc_const_binding_type<'tcx>(
     let mut guar = ty.visit_with(&mut collector).break_value();
 
     let ty_note = ty
-        .make_suggestable(tcx, false)
+        .make_suggestable(tcx, false, None)
```

Comment:
these parameters are quite hard to read now. it might make sense to pass in a struct that configures it instead of these opaque params (though I guess the IDE will display the parameter name so it is more readable)